src/app/lib/config/services/turkce.js

```diff
@@ -26,12 +26,12 @@ export const service = {
     defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/turkce.png',
     defaultImageAltText: 'BBC News Türkçe',
     dir: `ltr`,
-    externalLinkText: ', external',
-    imageCaptionOffscreenText: 'Image caption, ',
-    videoCaptionOffscreenText: 'Video caption, ',
-    audioCaptionOffscreenText: 'Audio caption',
-    defaultCaptionOffscreenText: 'Caption, ',
-    imageCopyrightOffscreenText: 'Image source, ',
+    externalLinkText: ', Dış Link',
+    imageCaptionOffscreenText: 'Fotoğraf altı yazısı: ',
+    videoCaptionOffscreenText: 'Video altyazısı: ',
+    audioCaptionOffscreenText: 'Ses dosyası altyazısı: ',
+    defaultCaptionOffscreenText: 'Altyazı ',
+    imageCopyrightOffscreenText: 'Kaynak: ',
```

Comment:
We shouldn't have the colons here `:` and in the caption offscreen text fields above, because in usage the text is followed by a comma anyway.

Reply:
I kind of assumed that the journalist knew what they were doing when they replaced the commas with colons. But are you saying we add another comma after it, which means for all the other services we have a double comma?

Reply:
Have thought about this some more, and you are of course right - I've changed them to commas.